activities/Measure.activity/js/activity.js

```diff
@@ -770,11 +779,11 @@ var app = new Vue({
 						doc.text(x, y, this.SugarL10n.get("Session") + ": " + i);
 						doc.setFontStyle("normal");
 						doc.text(this.SugarL10n.get("Mode") + ":     " + session.mode, x, y + 14);
-						if (this.log_interval == 300) {
+						if (session.interval == "300 second") {
```

Comment:
Hmmm. Seems dangerous to test a string specifically if the string could be localized